### DISCUSSION TITLE
version hammer db related linux Red Hat Enterprise Linux Server release 7.9 (Maipo)

### DISCUSSION THREAD

**Discussion by sandrogokmasinaga** [TPC-Council/HammerDB#663] [Q&A] [answered]
I will asking for version hammerDB related with linux Red Hat Enterprise Linux Server release 7.9 (Maipo)

**Comment by sm-shaw** [comment, marked as answer]
At the recent releases, we only build separate binaries for Red Hat 8.0 because this is a version that is supported by a number of the commercial databases. 

For any other Linux distribution, you can either use [Docker](https://hub.docker.com/r/tpcorg/hammerdb/tags) or as HammerDB is open source you build HammerDB from source! - this is very easy to do without coding skills and is explained [here](https://www.hammerdb.com/docs/ch01s13.html) and [here](https://www.hammerdb.com/blog/uncategorized/how-to-build-hammerdb-from-source/)

You only need to clone the HammerDB source, run one command as follows and the build will all happen automatically and create a distribution for the OS you built it on (this is exactly what we do for the release binaries)

`./Build-Linux.sh x64 Setup/HammerDB-Linux.bawt update`

A useful tip is that if you don't want to build HammerDB for all databases i.e. you don't want to source the 3rd party libraries to compile against for databases you are not using, then comment out the ones you don't want in the [setup](https://github.com/TPC-Council/HammerDB/blob/master/Build/Bawt-2.1.0/Setup/HammerDB-Linux.bawt) file before running the build. 

This way you can build HammerDB specifically for your databases on your Linux release, in this case Red Hat v7.9
